server/routes/testutils/appSetup.ts

```diff
@@ -29,6 +29,8 @@ const user = {
   displayName: 'John Smith',
 }
 
+export const flashProvider = jest.fn()
```

Comment:
Doesn't seem to be imported anywhere?

Reply:
Gets used in `server/routes/visitors.test.ts`

Reply:
Sorry, there were so many changes it didn't show and I assume it was a `package-lock.json` 😂